guides/micronaut-aws-secretsmanager/micronaut-aws-secretsmanager.adoc

```diff
@@ -75,25 +75,31 @@ You should see traces such as:
 [source, bash]
 ----
 curl localhost:8080
-XXXX
 ----
 
+```
```

Comment:
I am not sure that this is property asciidoc. I think they are Github fenced code blocks